INFO:     172.31.64.162:45558 - "GET /api/stats HTTP/1.1" 200 OK
INFO:     172.31.64.162:45558 - "GET /api/stats HTTP/1.1" 200 OK
INFO:     172.31.64.162:40436 - "POST /api/comment_campaigns HTTP/1.1" 200 OK
üîÑ –û—Ç–ø—Ä–∞–≤–∫–∞ –∫–æ–º–º–µ–Ω—Ç–∞—Ä–∏—è –æ—Ç –∞–∫–∫–∞—É–Ω—Ç–∞ 1 –≤ —á–∞—Ç @iirepresent, reply –∫ —Å–æ–æ–±—â–µ–Ω–∏—é 1747
üìù –ö–æ–º–º–µ–Ω—Ç–∞—Ä–∏–π: jh
üîÑ –ó–∞–ø—Ä–æ—Å –∫–ª–∏–µ–Ω—Ç–∞ –¥–ª—è –∞–∫–∫–∞—É–Ω—Ç–∞ 1
‚úÖ –ù–∞–π–¥–µ–Ω –∞–∫–∫–∞—É–Ω—Ç: VATREX (+1 424 491 6383)
–ù–∞–π–¥–µ–Ω —Ñ–∞–π–ª —Å–µ—Å—Å–∏–∏: sessions/session_14244916383.session
INFO:     172.31.64.162:40436 - "POST /api/comment_campaigns/12/start HTTP/1.1" 200 OK
‚úì –ö–ª–∏–µ–Ω—Ç –¥–ª—è –∞–∫–∫–∞—É–Ω—Ç–∞ 1 —É—Å–ø–µ—à–Ω–æ –ø–æ–¥–∫–ª—é—á–µ–Ω: VATREX
üîÑ –û—Ç–ø—Ä–∞–≤–∫–∞ –∫–æ–º–º–µ–Ω—Ç–∞—Ä–∏—è –æ—Ç –∞–∫–∫–∞—É–Ω—Ç–∞ 1 –≤ —á–∞—Ç @iirepresent, reply –∫ —Å–æ–æ–±—â–µ–Ω–∏—é 1747
üìù –ö–æ–º–º–µ–Ω—Ç–∞—Ä–∏–π: jh
‚ùå –û—à–∏–±–∫–∞ –æ—Ç–ø—Ä–∞–≤–∫–∏ –∫–æ–º–º–µ–Ω—Ç–∞—Ä–∏—è –ø–æ–¥ –ø–æ—Å—Ç –∞–∫–∫–∞—É–Ω—Ç–æ–º 1: Telegram says: [400 CHAT_ADMIN_REQUIRED] - The method requires chat admin privileges (caused by "messages.SendMessage")
üîÑ –ü—Ä–æ–±—É–µ–º –Ω–∞–π—Ç–∏ –≥—Ä—É–ø–ø—É –æ–±—Å—É–∂–¥–µ–Ω–∏–π –¥–ª—è –∫–∞–Ω–∞–ª–∞ @iirepresent
üì¢ –ù–∞–π–¥–µ–Ω–∞ –≥—Ä—É–ø–ø–∞ –æ–±—Å—É–∂–¥–µ–Ω–∏–π: -1001924490125
‚úÖ –ö–æ–º–º–µ–Ω—Ç–∞—Ä–∏–π –æ—Ç–ø—Ä–∞–≤–ª–µ–Ω –≤ –≥—Ä—É–ø–ø—É –æ–±—Å—É–∂–¥–µ–Ω–∏–π –∞–∫–∫–∞—É–Ω—Ç–æ–º 1
‚ùå –ò—Å–∫–ª—é—á–µ–Ω–∏–µ –ø—Ä–∏ –æ—Ç–ø—Ä–∞–≤–∫–µ –∫–æ–º–º–µ–Ω—Ç–∞—Ä–∏—è: (sqlite3.OperationalError) table comment_logs has no column named chat_id
[SQL: INSERT INTO comment_logs (campaign_id, account_id, chat_id, message_id, comment, status, error_message, sent_at) VALUES (?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (12, 1, '@iirepresent', 1747, 'jh', 'success', None, '2025-08-14 12:56:33.505079')]
(Background on this error at: https://sqlalche.me/e/20/e3q8)
‚ùå –û—à–∏–±–∫–∞ –≤ –∫–∞–º–ø–∞–Ω–∏–∏ –∫–æ–º–º–µ–Ω—Ç–∏—Ä–æ–≤–∞–Ω–∏—è 12: This Session's transaction has been rolled back due to a previous exception during flush. To begin a new transaction with this Session, first issue Session.rollback(). Original exception was: (sqlite3.OperationalError) table comment_logs has no column named chat_id
[SQL: INSERT INTO comment_logs (campaign_id, account_id, chat_id, message_id, comment, status, error_message, sent_at) VALUES (?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (12, 1, '@iirepresent', 1747, 'jh', 'success', None, '2025-08-14 12:56:33.505079')]
(Background on this error at: https://sqlalche.me/e/20/e3q8) (Background on this error at: https://sqlalche.me/e/20/7s2a)
